.github/workflows/release_gem.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags:
+      - '**'
```

Comment:
Nit: This still could be just a single `'*'`

Reply:
I would stick to the recommendation of the [actions README](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags)

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#patterns-to-match-branches-and-tags

Reply:
Ah, didn't realize the difference :) I doubt we want tags with `/` in a name released, but sure, I'm fine with it! :+1: